syncd/VendorSai.cpp

```diff
@@ -95,6 +99,10 @@ sai_status_t VendorSai::initialize(
         }
 
         m_apiInitialized = true;
+
+#ifdef MDIO_ACCESS_USE_NPU
+        MdioIpcServer::setSwitchMdioApi(m_apis.switch_api);
+#endif
```

Comment:
please remove this from Vendor SAI, this is not apart of there, should be set in Constructor

Reply:
See reply to preceding comment.

Reply:
virtual functions mdioRegRead/mdioRegWrite are declared in SaiInterfaces class and are overridden in VendorSai class. MdioIpcServer::setSwitchMdioApi is no longer needed.